src/main/java/org/motechproject/newebodac/service/ImportService.java

```diff
@@ -195,9 +205,9 @@ private void handleVaccineeEntity(Map<String, Object> cleanRow) {
   }
 
   private void addFieldValueMapErrors(ICsvMapReader csvMapReader, Map<Integer, String> errorMap,
-      String cellValue, CsvField csvField) {
+      String cellValue, CsvField csvField, Map errorValues) {
```

Comment:
you can use `Map<String, ?> errorValues`, also rename this, it's more like available values map or something like this